mlflow/tracking/_tracking_service/utils.py

```diff
@@ -190,19 +190,28 @@ def _get_databricks_uc_rest_store(store_uri, **_):
     )
 
 
+global _tracking_store_registry
 _tracking_store_registry = TrackingStoreRegistry()
-_tracking_store_registry.register("", _get_file_store)
-_tracking_store_registry.register("file", _get_file_store)
-_tracking_store_registry.register("databricks", _get_databricks_rest_store)
-_tracking_store_registry.register(_DATABRICKS_UNITY_CATALOG_SCHEME, _get_databricks_uc_rest_store)
 
-for scheme in ["http", "https"]:
-    _tracking_store_registry.register(scheme, _get_rest_store)
 
-for scheme in DATABASE_ENGINES:
-    _tracking_store_registry.register(scheme, _get_sqlalchemy_store)
+def register_tracking_stores():
```

Comment:
can we mark it as internal method ? like `_register_tracking_stores`

Reply:
Good point. Changed.